tarpc/examples/readme.rs

```diff
@@ -77,13 +76,9 @@ async fn run() -> io::Result<()> {
     Ok(())
 }
 
-fn main() {
-    tarpc::init(tokio::executor::DefaultExecutor::current().compat());
-
-    tokio::run(
-        run()
-            .map_err(|e| eprintln!("Oh no: {}", e))
-            .boxed()
-            .compat(),
-    );
+#[runtime::main]
```

Comment:
I'm surprised this doesn't need to be `runtime::main(Tokio)`? Because `bincode_transport` uses tokio tcp.